src/harness/testcases/testdata/wdb_2/WDB_2_CPI_Database.csv

```diff
@@ -0,0 +1,10 @@
+cpiId,cpiName,mailingAddress,legalAddress,email,phone,trainingProgram,completionDate,trainingProgramAdministrator,licenseStartDate,licenseEndDate ,status,publicKeyIdentifier 

```

Comment:
This is not the correct format; please refer to the meeting minutes and the CPI database spec for the latest information on this.

Reply:
this file is removed as suggested before.